src/panels/media-browser/ha-bar-media-player.ts

```diff
@@ -227,7 +227,9 @@ export class BarMediaPlayer extends SubscribeMixin(LitElement) {
         })}
         @click=${this._openMoreInfo}
       >
-        ${mediaArt ? html`<img src=${this.hass.hassUrl(mediaArt)} />` : ""}
+        ${mediaArt
+          ? html`<img alt="" src=${this.hass.hassUrl(mediaArt)} />`
```

Comment:
This is not ideal, but I am presuming that
1. the art is not large and prominent, and
2. there is no suitable alternative text gathered for the entity (the docs are pretty lacking regarding media)

Reply:
So this is the album cover / radio art etc. If we would want a description, we could get the media title and do something like `Album art for ${mediaTitleClean || mediaDescription}`. But screen readers then end up reading the media title twice (as it's the next element too)

Reply:
Nah, it should remain `""` if the best we have is just the title.  Something better would have to come from the source of the art.  I just figured I'd pose the question in case it's there but we don't take advantage.